go/pkg/cas/upload_test.go

```diff
@@ -94,13 +96,81 @@ func TestFS(t *testing.T) {
 		inputs              []*UploadInput
 		wantScheduledChecks []*uploadItem
 		wantErr             error
+		wantErrContain      string
```

Comment:
instead of a `wantErrContain`, we can make a `var ErrDanglingSymlink = errors.New("symlink's target was filtered out")` and return `errors.Wrap(ErrDanglingSymlink, "path: %v, target: %v")`.

I generally think sentinels are nicer than relying on strings, as to not allow eg the same string to match multiple errors.

Reply:
I had to name it `ErrFilteredSymlinkTarget` though, because `ErrDanglingSymlink` looks like result of `!AllowDanglingSymlink`, in which case we actually don't return it.

Reply:
lgtm